### PR TITLE
gnupatch: new package at 2.7.6

### PR DESCRIPTION
**Pull request by cemkeylan**
## Description of package

gnupatch is nice if you are doing complex patching operations. It is faster
and knows what `/dev/null` is.

## Checklist

- [X] Latest upstream version.
- [X] I agree to maintain this package (**required**).
- [X] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/gnupatch/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+http://ftp.gnu.org/gnu/patch/patch-2.7.6.tar.xz
```
Comment (dylanaraps) [inline community/gnupatch/sources]: No https?

Comment (cemkeylan) [reply community/gnupatch/sources]: Oh sorry must have forgotten

**Comment by cemkeylan** [comment]
Fixed it

**Comment by a-schaefers** [comment]
we already have gnupatch and it is called "patch"

**Comment by cemkeylan** [comment]
I have also replied on the IRC, but what is your point, exactly @a-schaefers ?

**Pull request merged**